ui/src/shared/graphs/helpers.js

```diff
@@ -26,7 +26,7 @@ export const darkenColor = colorStr => {
   return `rgb(${color.r},${color.g},${color.b})`
 }
 
-// Bar Graph code below is from http://dygraphs.com/tests/plotters.html
+// Bar Graph code below is adapted from http://dygraphs.com/tests/plotters.html
 export const multiColumnBarPlotter = e => {
```

Comment:
comments in general in this section would be helpful for review and later debugging